ui-lib/src/main/res/ui/advanced_settings/values/strings.xml

```diff
@@ -4,7 +4,7 @@
     <string name="advanced_settings_export">Export Private Data</string>
     <string name="advanced_settings_choose_server">Choose a Server</string>
     <string name="advanced_settings_currency_conversion">Currency Conversion</string>
-    <string name="advanced_settings_coinbase">Buy ZEC with Coinbase</string>
+    <string name="advanced_settings_coinbase">Buy %s with Coinbase</string>
```

Comment:
Should we use xliff for the variable replacement here instead?

Reply:
I think its overkill no ? %s is quite sufficient

Reply:
It would work, of course, but we use the widely spread xliff standard to replace variables across all our string files for a unit approach regardless of the variable type. Let's keep it here as well, please.